packages/core/admin/admin/src/pages/MarketplacePage/components/NpmPackageCard/PackageStats.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Typography } from '@strapi/design-system/Typography';
+import { Icon } from '@strapi/design-system/Icon';
+import { Divider } from '@strapi/design-system/Divider';
+import { Stack } from '@strapi/design-system/Stack';
+import Github from '@strapi/icons/Github';
+import Download from '@strapi/icons/Download';
+import Star from '@strapi/icons/Star';
+
+const VerticalDivider = styled(Divider)`
+  width: 0.75rem;
+  transform: rotate(90deg);
+`;
+
+const PackageStats = ({ githubStars, weeklyDownloads }) => {
+  return (
+    <Stack horizontal spacing={1}>
+      {!!githubStars && (
+        <>
+          <Icon as={Github} height="0.75rem" width="0.75rem" />
+          <Icon as={Star} height="0.75rem" width="0.75rem" color="warning500" />
+          <p aria-label={`This package was starred ${githubStars} on GitHub`}>
```

Comment:
Every string in the UI needs to be translate-able and therefore needs to go through `formatMessage()`. I think you can check the other components on how it is done.

Reply:
sure, noted!